frontend/src/app/auth/auth.service.ts

```diff
@@ -1,140 +1,162 @@
 import { Injectable } from '@angular/core';
+import { HttpHeaders } from '@angular/common/http';
 import { Router } from '@angular/router';
-import { BehaviorSubject } from 'rxjs';
-import { AUTH_CONFIG } from './auth.config';
-import * as auth0 from 'auth0-js';
-import { ENV } from '../core/env.config';
+import {
+  Auth0DecodedHash as DecodedHash,
+  Auth0UserProfile as User,
+  Management,
+  WebAuth,
+} from 'auth0-js';
```

Comment:
I like this a lot

Reply:
Not sure if sarcastic........ :thinking: